CityOfDetroit/bloom
# Remove Application landing pages

# Pull Request Template

## Issue

Addresses #159 


## Description

This PR removes any pages associated with Applications.  UI components related to applications still exist, but the landing pages created upon launch are deleted.  Should the need arise to reenable Applications, the files and landing pages can be added again from upstream.

## Type of change


## How Can This Be Tested/Reviewed?

Visit any active listing and try to follow the Apply Now link.  The landing page will show a 404 Not Found error.  Additionally for any listing you know the listing ID, try visiting the following:
http://localhost:3000/applications/start/choose-language?listingId=**INSERT LISTING ID HERE**
This should also result in a 404 Not Found error.


![image](https://user-images.githubusercontent.com/82653098/128872502-7aa16eb8-808e-48b6-b33b-fcdcc3c27622.png)


## Checklist:
